code/modules/events/patient_zero.dm

```diff
@@ -0,0 +1,55 @@
+/datum/round_event_control/patient_zero
+	name = "Patient Zero"
+	typepath = /datum/round_event/ghost_role/patient_zero
+	weight = 5
+
+	min_players = 10
+
+/datum/round_event/ghost_role/patient_zero
+	announceWhen	= 400
+
+	minimum_required = 1
+	role_name = "patient zero"
+
+	// 50% chance of being incremented by one
+	var/spawncount = 1
+	fakeable = TRUE
+
+/datum/round_event/ghost_role/patient_zero/announce(fake)
+	priority_announce("Unidentified lifesigns detected coming aboard [station_name()]. Secure any exterior access, including ducting and ventilation.", "Lifesign Alert", 'sound/ai/aliens.ogg')
+
+/datum/round_event/ghost_role/patient_zero/spawn_role()
+	var/list/candidates = get_candidates(ROLE_ALIEN, null, ROLE_ALIEN)
+	if(!candidates.len)
+		return NOT_ENOUGH_PLAYERS
+
+	var/mob/dead/selected = pick(candidates)
+
+	var/list/spawn_locs = list()
+	for(var/obj/effect/landmark/carpspawn/C in GLOB.landmarks_list)
```

Comment:
You are spawning them in space, wouldn't maints make more sense?

Reply:
I see it as just a zombie floating away through space, which was lost (or thrown away) from another space ship/station. Hence the space spawn/being thrown at the station...